debugger/terminal/terminal_other.go

```diff
@@ -11,8 +11,8 @@ import (
 	"sync"
 	"syscall"
 
+	"github.com/earthly/earthly/conslogging"
 	"github.com/earthly/earthly/debugger/common"
-	"github.com/earthly/earthly/logging"
```

Comment:
Any chance we can get rid of this package entirely?

Reply:
It's used by the shellrepeater for logging; and is designed for structured-logging for use in servers. Maybe we should rename it to `structuredlogging`?

Reply:
Ok no worries. Maybe something shorter, like `slog`? In any case, this is unrelated to this PR.

Reply:
I'll open a new PR for that rename.